doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -39,7 +39,8 @@
 - TargetingSessionBagListener - changed the signature of `__construct`.
 - Removed deprecated methods `getObject()` and `setObject()` on the classes `Pimcore\Model\Document\Link` and `Pimcore\Model\DataObject\Data\Link`, please use `getElement()` and `setElement()` instead.
 - Removed deprecated `Pimcore\Db\ConnectionInterface` interface, `Pimcore\Db\Connection` class and `Pimcore\Db\PimcoreExtensionsTrait` trait.
-- Removed deprecated `Pimcore\Routing\Dynamic\DocumentRouteHandler::addDirectRouteDocumentType()` method, please use the `pimcore.routing.direct_route_document_types` config instead.
+- Removed deprecated `Pimcore\Routing\Dynamic\DocumentRouteHandler::addDirectRouteDocumentType()` method, please use the `pimcore.documents.type_definitions.map.%document_type%.direct_route` config instead.
+- Moved `pimcore.documents.valid_tables` to the type definition it self `pimcore.documents.type_definitions.map.%document_type%.valid_table`
```

Comment:
```suggestion
- Refactored `pimcore.documents.valid_tables` to be an option under each document type definition (eg. `pimcore.documents.type_definitions.map.%document_type%.valid_table`) instead of an own array node
```